src/style.css

```diff
@@ -0,0 +1,51 @@
+* {
+  margin: 0;
+}
+
+.stars {
+  list-style: none;
+  padding: 0;
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  margin: 0 2px;
```

Comment:
just use like this to add an indent between stars

.stars__star:not(:last-child) {
  margin-top: 4px;
}